core/src/components/header/header.stories.tsx

```diff
@@ -71,7 +71,8 @@ const Template = () =>
     </tds-header-launcher>
 
 
-    <tds-header-brand-symbol slot="end" link-href="https://scania.com" aria-label="Scania - red gryphon on blue shield">
+    <tds-header-brand-symbol slot="end" aria-label="Scania - red gryphon on blue shield">
```

Comment:
The aria label should be moved to the `<a>` tag since that is the element with the image.